app/(afterLogin)/(deal)/deals/manage-deals/_components/RecentActivities.tsx

```
@@ -0,0 +1,164 @@
+  id: string;
+  type: string;
+  description: string;
+  date: string;
+  status?: 'completed' | 'pending' | 'in-progress';
+}
+
+interface RecentActivitiesProps {
+  dealId: string;
+}
+
+export default function RecentActivities({}: RecentActivitiesProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)